addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -58,10 +58,18 @@
         </DatiTrasmissione>
         <CedentePrestatore>
             <DatiAnagrafici>
-                <IdFiscaleIVA>
+                <IdFiscaleIVA t-if="seller.vat and in_eu(seller)">
                     <IdPaese t-esc="get_vat_country(seller.vat)"/>
                     <IdCodice t-esc="get_vat_number(seller.vat)"/>
                 </IdFiscaleIVA>
+                <IdFiscaleIVA t-if="seller.vat and not in_eu(seller)">
+                    <IdPaese t-esc="seller.country_id.code"/>
+                    <IdCodice t-esc="'OO99999999999'"/>
+                </IdFiscaleIVA>
+                <IdFiscaleIVA t-if="not seller.vat and seller.country_id.code != 'IT'">
```

Comment:
@lordkrandel I wonder if this t-if is a realistic use case, but otherwise lgtm.

Reply:
Yes, it is, the ticket I talk about in the description is about this.
We also have to investigate this https://www.odoo.com/web#id=3044072&cids=1&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form

Reply:
Before, wrong
![image](https://user-images.githubusercontent.com/1665365/200836705-4923f92f-003a-4f5f-be92-5712b0906f30.png)
After, correct
![image](https://user-images.githubusercontent.com/1665365/200836749-47d34612-ab08-468e-b003-cebe9c0c4811.png)

Reply:
I've made a simple vendor bill with a US partner (GMass, which is Google)